File: docs/snippets/python/agent-message_main.py
# from microsoft.agents.xml.models import AgentMessage, TextContent
# from microsoft.agents.xml.serialization import MessageSerializer
#
# # Create agent response
# message = AgentMessage(
#     role="assistant",
#     agent_id="agent-456",
#     message_id="msg-789",
#     contents=[
#         TextContent(text="The current weather in Seattle is 55°F and partly cloudy.")
#     ]
# )
#
# serializer = MessageSerializer()
# xml_output = serializer.serialize(message)
# print(xml_output)

File: docs/snippets/csharp/agent-message_main.cs
// using Microsoft.Agents.Xml.Generated.Models;
// using Microsoft.Agents.Protocol.Xml;

// Create agent response
var message = new AgentMessage
{
    AgentId = "agent-456",
    MessageId = "msg-789",
    Contents = new List<AIContent>
    {
        new TextContent { Text = "The current weather in Seattle is 55°F and partly cloudy." }
    }
};

var serializer = new MessageSerializer();
var xmlOutput = serializer.Serialize(message);
Console.WriteLine(xmlOutput);